planetscale.gemspec

```diff
@@ -35,7 +35,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.add_development_dependency 'bundler', '~> 2.2'
+  spec.add_development_dependency 'bundler', '~> 2'
```

Comment:
I loosened this pin to major version only because the official `ruby:2.7` Docker images ship with `bundler = 2.1.8`.
CI passed.. is there anything wasp nest I'm stepping on that I don't see from downgrading a minor version?